pkg/state/errors/errors.go

```diff
@@ -9,6 +9,8 @@ import (
 )
 
 var (
+	BrokerInUseError    error = fmt.Errorf("broker in use")
+	NotTenantError      error = fmt.Errorf("not tenant found")
```

Comment:
NotTenantError reads oddly.   I think you mean TenantNotFoundError

Reply:
Agreed, fixed.